pvr.hts/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.hts"
-  version="4.0.16"
+  version="4.0.17"
```

Comment:
Maybe we should use `4.1.0` instead since this implements a new verison the PVR API?